### PR TITLE
Add job configs for the storage migrator repository

### PR DESCRIPTION
**Pull request by caesarxuchao**
And the Makefile of the repo will be at: https://github.com/kubernetes-sigs/kube-storage-version-migrator/pull/3/commits/bfff573531572f87b1d7523bb7333001b8e88e19

(PS: I tried to follow the README "Please test the job on your local workstation before creating a PR", but not sure what service account to use to launch GCE/GKE clusters)

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+presubmits:
+  kubernetes-sigs/kube-storage-version-migrator:
+  - name: pull-migrator-test
```
Comment (BenTheElder) [inline config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: this looks like a good candidate for https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md which are replacing bootstrap.py (slowly!)

basically just:
- add `decorate: true` below the `name: pull-migrator-test`
- add a `path_alias: my.go/import-path` below that if your go import path is not github.com/kubernetes-sigs/kube-storage-version
- drop the label / preset-service-account
- on the container below, replace the args with:
```
command:
- make
- test
```

Comment (caesarxuchao) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: Done. Can you take another look? Thanks.

File: config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    decorate: true
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180912-7a027de0c-master
+        args:
```
Comment (krzyzacy) [inline config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: I think you can drop all args section

Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: yeah, after dropping the `args:` section this should be good to go

**Comment by krzyzacy** [comment]
also enable `trigger` plugin for your repo around https://github.com/kubernetes/test-infra/blob/master/prow/plugins.yaml#L472-L509 :-)

File: testgrid/config.yaml [hunk 3]
```diff
@@ -2832,6 +2832,9 @@ test_groups:
 - name: cloud-provider-aibaba-cloud-e2e-conformance-release-v1.8
   gcs_prefix: k8s-conformance-alibaba/cloud-provider-alibaba-cloud/e2e-conformance-release-v1.8
 
+# kube-storage-version-migrator
+- name: pull-kube-storage-version-migrator-test
+  gcs_prefix: kubernetes-jenkins/logs/pull-kube-storage-version-migrator-test
```
Comment (krzyzacy) [inline testgrid/config.yaml]: this should be

```
gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-migrator-test
```

File: config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml (hunk 2 above)
Comment (caesarxuchao) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: Done.

File: testgrid/config.yaml (hunk 3 above)
Comment (caesarxuchao) [reply testgrid/config.yaml]: Thanks. I changed the name in the job config.

File: config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml (hunk 2 above)
Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: I don't think it pushed?

File: prow/plugins.yaml [hunk 4]
```diff
@@ -508,6 +508,12 @@ plugins:
   kubernetes-sigs/cri-o:
   - trigger
 
+  kubernetes-sigs/kube-storage-version-migrator:
+  - assign
+  - label
+  - lgtm
```
[the remark this answers is not in the document]
Comment (caesarxuchao) [reply prow/plugins.yaml]: Removed.

**Pull request merged**